src/contracts/utils.ts

```diff
@@ -1,5 +1,6 @@
 import assert from "assert";
-import { Event } from "ethers";
+// eslint-disable-next-line no-restricted-imports
```

Comment:
Is this a required import?

Reply:
I'm a little unsure on this specifically - it does omit most of the fields that we don't care about within `Event`, but the downstream `poolClient` subsequently converts up to an `Event-ish` type.

As background, this is basically legacy code that was imported from the UMA repo a while ago. We depend on it in the FE, but not in the bots. This makes it a little trickier for me to test, so I took the conservative route here (...admittedly, last time I touched something in the `poolClient` it subtly broke the `Pool` page in the FE).

Reply:
@james-a-morris Also on the above, the fact that this is legacy-leaning code and is only used by the FE makes me wonder if it could be a candidate for relocation to the FE. wdyt?